lib/collection/request-body.js

```diff
@@ -17,26 +17,26 @@ _.inherit((
      * @constructor
      * @extends {PropertyBase}
      *
-     * @param {Object} options
+     * @param {*} options
      */
     RequestBody = function PostmanRequestBody (options) {
         // this constructor is intended to inherit and as such the super constructor is required to be executed
         RequestBody.super_.apply(this, arguments);
-        if (!options) { return; } // in case definition object is missing, there is no point moving forward
 
-        this.update(options);
+        options && this.update(options); // only set the request body if options are truthy
     }), PropertyBase);
 
 _.assign(RequestBody.prototype, /** @lends RequestBody.prototype */ {
     /**
      * Set the content of this request data
      *
-     * @param {Object} options
+     * @param {*} options
      */
     update: function (options) {
-        if (!options.mode) { return; } // need a valid mode @todo raise error?
+        if (!options) { return; }
 
-        var mode = RequestBody.MODES[options.mode.toString().toLowerCase()] || RequestBody.MODES.raw,
+        var rawMode = options.mode && options.mode.toString().toLowerCase(),
```

Comment:
`rawMode` is ambiguous name as it seems one is talking about "raw" mode. This variable is not needed since we can do:

```
mode = RequestBody.MODES[options.mode && options.mode.toString().toLowerCase()] || RequestBody.MODES.raw,
```